### PR TITLE
fix issue where errors were not being caught

### PR DESCRIPTION
**Pull request by xjamundx**
The try/catch here was not working as expected. I found the following change would correctly handle the errors.

Caveat, I'm not a streams pro AT ALL. I straight up stole this from here:
http://stackoverflow.com/questions/21771220/error-handling-with-node-js-streams

Let me know if you have any other thoughts or ideas!

### CODE REVIEW

**Comment by xjamundx** [comment]
Added a test to ensure that it works (and verified the test failed before)

File: plugins/localize.js [hunk 1]
```diff
@@ -35,11 +35,11 @@ module.exports.localize = function localize(srcFile, propFile, cb) {
...
-    try {
-        localizr.createReadStream(opt).pipe(out);
-    } catch (err) {
-        cb(err);
-    }
+
+    var readStream = localizr.createReadStream(opt);
+    var writeStream = readStream.pipe(out);
+    readStream.on('error', cb);
+    writeStream.on('error', cb);
```
Comment (aredridel) [inline plugins/localize.js]: This seems reasonable. Can be done without temporary variables, but works.

Comment (xjamundx) [inline plugins/localize.js]: Agreed, but it's sort of harder to read:
```javascript
localizr.createReadStream(opt).on('error', cb)
           .pipe(out).on('error', cb)
````

**Comment by pvenkatakrishnan** [comment]
Also jamund, did you pull in the latest from master ? I dont see some changes in master on this file on the changed one ?

**Comment by pvenkatakrishnan** [comment]
for example fix to check if templates exist in this commit ..https://github.com/krakenjs/kraken-devtools/commit/994717c480b7a2246dfdf97838b3de0d9eb08b84

**Comment by pvenkatakrishnan** [comment]
ups sorry closed by mistake.

**Comment by xjamundx** [comment]
let me rebase, then the tests might stop failing  :)

**Comment by xjamundx** [comment]
Unclear why the thing is failing, tests pass locally

**Comment by pvenkatakrishnan** [comment]
@xjamundx the tests are failing on travis due to an unrelated issue in the sass compiler which is incompatible between 0.10 and 0.11 of node. Travis runs all our jobs against both versions and hence failing.  There is an issue open in node-sass which with next version should fix to work seamlessly. Don't worry about that.

**Comment by pvenkatakrishnan** [comment]
if you;d like more info on that issue https://github.com/krakenjs/kraken-devtools/issues/35

**Comment by pvenkatakrishnan** [comment]
Merging this in.

**Comment by xjamundx** [comment]
thanks!

**Pull request merged**